### PR TITLE
Prepare release v0.6.7

### PR DESCRIPTION
**Pull request by tunetheweb**
### Brief summary of the change made
We've a lot of changes made since 0.6.6 so think it's time to prepare for 0.6.7!

There's still a few PRs almost ready to merge.
@sti0 , @barrywhart , @jpers36 , @leamingrad - let me know if you if you think you'll have your PRs ready to go in next day or so, by commenting below.

And anyone else, do let us know if working on something that you think is almost ready to go!

### Are there any other side effects of this change that we should be aware of?
Nope - unless you like bugs that is!
Though I did fix some bad links on 0.6.6 CHANGELOG.

### Pull Request checklist
- [x] Please confirm you have completed any of the necessary steps below.

- Included test cases to demonstrate any code changes, which may be one or more of the following:
  - `.yml` rule test cases in `test/fixtures/rules/std_rule_cases`.
  - `.sql`/`.yml` parser test cases in `test/fixtures/parser` (note YML files can be auto generated with `python test/generate_parse_fixture_yml.py` or by running `tox` locally).
  - Full autofix test cases in `test/fixtures/linter/autofix`.
  - Other.
- Added appropriate documentation for the change.
- Created GitHub issues for any relevant followup/future enhancements if appropriate.

### CODE REVIEW

**Comment by barrywhart** [comment]
My PR is ready for review, just waiting for more 👀.

**Comment by sti0** [comment]
#1491 is ready for review, too 😃

**Comment by jpers36** [comment]
#1486 has been updated!

**Comment by leamingrad** [comment]
#1483 Is good to go if needed

**Comment by tunetheweb** [comment]
> #1483 Is good to go if needed

Added to this PR!

**Comment by barrywhart** [comment]
Ok, #1502 is merged. If we're releasing 0.6.7 today, I think the other one (#1496, the dbt PR) probably won't be ready. I'm still wrestling with some fussy pathing issues, also my initial tester is in Australia and is now asleep. 😄 I asked for other reviewers in the "help" channel on Slack. I don't want to rush this one, because this is the first time I've worked on the dbt templater.

**Comment by tunetheweb** [comment]
OK I think we've enough for now. Preparing the release...

File: CHANGELOG.md [hunk 1]
```diff
@@ -9,41 +9,89 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 Note: Changes are now automatically tracked in [GitHub](https://github.com/sqlfluff/sqlfluff/releases) and will be copied in here on each release (please remember to update the issues and contributors to links!). There is no need to manually edit this file going forward.
 -->
 
+## [0.6.7] - 2021-10-04
+
+Lots of bug fixes to our rules (particularly the autofixer), plus good improvements to Exasol, Snowflake, and T-SQL, dialects amongst others! Plus we added Hive and SQLite as supported dialects!
```
Comment (barrywhart) [inline CHANGELOG.md:14]: Re: The autofixer, maybe mention that many of the fixes address improve handling of Jinja and dbt templating.

**Review by barrywhart — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (tunetheweb) [reply CHANGELOG.md]: Good idea. Changed it to:

> Lots of fixes to our rules (particularly when running `sqlfluff fix`, and particularly for Jinja and DBT templates). We also have good improvements to Exasol, Snowflake, and T-SQL dialects amongst others. Plus we added Hive and SQLite as supported dialects!

**Pull request merged**